### ISSUE TITLE
Hovering over Undo/Redo arrows should give a hint of *which* tags it is talking about

### ISSUE THREAD

**Issue by jidanni** [openstreetmap/iD#7684]
If we hover over the Undo/Redo arrow,
it might say we "Changed tags".

That is great, but we need a little hint of which tags they were,
before we know if we want to undo / redo that step.
Mentioning just one tag, with an ellipsis, would be good enough for starters. Thanks.

No it is not always clear from looking at the left panel.

**Comment by quincylvania** [comment]
I agree we could have more relevant messages for certain types of tag changes, like "Changed feature type" or "Changed Name value". But I don't think iD should mention specific tags, unless perhaps the mapper was using the raw tag editor.

**Comment by jidanni** [comment]
(Sometimes it is so worrisome, that one opts to just save, and then have a look at what was saved and  clean up (edit) anything that one didn't intend.)
